src/main/java/lotto/utils/Splitter.java

```diff
@@ -0,0 +1,18 @@
+package lotto.utils;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Splitter {
+
+    private Splitter() {
+        throw new IllegalStateException("Utility Class!!");
+    }
+
+    public static List<Integer> splitToNumberBy(String string, String delimiter) {
+        string = string.replace(" ", "");
+        int[] numbers = Arrays.stream(string.split(delimiter)).mapToInt(Integer::parseInt).toArray();
+        return Arrays.stream(numbers).boxed().collect(Collectors.toList());
```

Comment:
가독성을 위해 줄바꿈을 하고 아래처럼 합칠수 있을것 같아요.
```suggestion
        return Stream.of(string.split(delimiter))
                .mapToInt(Integer::parseInt)
                .boxed()
                .collect(Collectors.toList());